config.yaml

```diff
@@ -18,9 +18,9 @@ options:
       in https://github.com/dexidp/dex#connectors
   static-username:
     type: string
-    default: ''
+    default: 'admin'
```

Comment:
Will this break upgrade?

Reply:
I think we park this thing till next release